cwe_checker_rs/src/pcode/expressions.rs

```diff
@@ -122,6 +122,35 @@ pub struct Expression {
     pub input2: Option<Variable>,
 }
 
+impl From<Expression> for IrExpression {
+    /// Translates a P-Code expression into an expression of the internally used IR if possible.
+    /// Panics if translation is not possible,
+    /// e.g. for `LOAD`, `STORE` and and expressions that need the size of the output variable to be defined.
```

Comment:
I think there is an expression type missing

Reply:
You are right. I will clarify the doc comment!